### DISCUSSION TITLE
Shouldn't `rmh-elfeed-org-files` go into `/etc` directory?

### DISCUSSION THREAD

**Discussion by snowman** [emacscollective/no-littering#184] [Support] [answered]
The `rmh-elfeed-org-files` are the list of configuration files where [elfeed-org](https://github.com/remyhonig/elfeed-org) find feeds.

**Comment by tarsius** [comment, marked as answer]
Yes. The name of the variable shouldn't be prefixed with `rmh-` though. We should use `(etc "elfeed-org/")`.
